securedrop/tests/client_test.py

```diff
@@ -57,10 +57,19 @@ def test_aac_initial_ask_to_register_empty_input(self):
                         with self.assertRaises(RuntimeError):
                             Client(sd_filename).login()
 
+    def test_initial_ask_to_register_password_too_short(self):
```

Comment:
Test function name should follow similar convention to others with prefix being `test_aai` where `i` is the `ith` test to run. I couldn't find a better way to ensure deterministic test order -- I know it's clunky. This is the perfect spot for the function, so it should be `test_aad`. The tests below will need to have their prefixes adjusted accordingly to maintain the same order after the insertion, so replace the previous `test_aad`with `test_aae` and so on.

Reply:
Will do.

Reply:
done